packages/manager/modules/dbaas-logs/src/logs/detail/streams/subscriptions/streams-subscriptions.service.js

```diff
@@ -0,0 +1,60 @@
+export default class LogsStreamsSubscriptionsService {
+  /* @ngInject */
+  constructor($http, iceberg) {
+    this.$http = $http;
+    this.iceberg = iceberg;
+  }
+
+  /**
+   * Retrieve list of stream's subscription with pagination, sorts, filters etc.
+   * @param serviceName string LDP service name
+   * @param streamId string LDP stream UUID
+   * @param offset int Offset to start from
+   * @param pageSize int Number of results to retrieve from API
+   * @param sort string Name of field to sort from
+   * @param filters Array List of Iceberg filters to apply
+   * @return {Object}
+   */
+  getPaginatedStreamSubscriptions(
+    serviceName,
+    streamId,
+    offset = 0,
+    pageSize = 25,
+    sort = { name: 'nbArchive', dir: 'desc' },
+    filters = null,
+  ) {
+    let res = this.iceberg(
+      `/dbaas/logs/${serviceName}/output/graylog/stream/${streamId}/subscription`,
+    )
+      .query()
+      .expand('CachedObjectList-Pages')
+      .limit(pageSize)
+      .offset(offset)
+      .sort(sort.name, sort.dir);
+    if (filters !== null) {
+      filters.forEach((filter) => {
+        res = res.addFilter(filter.name, filter.operator, filter.value);
+      });
+    }
```

Comment:
```suggestion
   
      filters?.forEach((filter) => {
        res = res.addFilter(filter.name, filter.operator, filter.value);
      });
    
```

Reply:
Tried it, functionally it works, but it's rejected by `eslint`:

```
manager/packages/manager/modules/dbaas-logs/src/logs/detail/streams/subscriptions/streams-subscriptions.service.js
  34:5  error  Expected an assignment or function call and instead saw an expression  no-unused-expressions
```
This `eslint` failure is pretty silly, as it's the same as the original code.

I kept it as-is to avoid this `eslint` failure, and also for an [homogenization purpose with other code in this module](https://github.com/search?q=repo%3Aovh%2Fmanager%20if%20%7B%20filters.forEach((filter)%20%3D%3E%20%7B%20res%20%3D%20res.addFilter(filter.name%2C%20filter.operator%2C%20filter.value)%20%3B%20%7D)%3B%20%7D&type=code  )